Documentation/dev-guide/apispec/swagger/rpc.swagger.json

```diff
@@ -2177,7 +2184,7 @@
           "format": "uint64"
         },
         "raftAppliedIndex": {
-          "description": "appliedIndex is the current raft applied index of the responding member.",
+          "description": "raftAppliedIndex is the current raft applied index of the responding member.",
```

Comment:
can we fix this in a separate pr? thanks. i want to keep the commit as clean as possible.

Reply:
I didn't notice this change. Will have a subsequent PR for this. Not sure why this didn't get updated in the last PR when I did the proto gen.